<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invoicing</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Invoicing</string>
    </attribute>
    <widget class="MyCalendar" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="firstDayOfWeek">
      <enum>Qt::Saturday</enum>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="invoice_scrollArea">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>461</width>
       <height>281</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>459</width>
        <height>279</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>341</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="invoice_number_lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Invoice No.</string>
      </property>
     </widget>
     <widget class="QComboBox" name="payment_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>72</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Bank</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cash</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="add_contact_Button1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Contact</string>
      </property>
     </widget>
     <widget class="QPushButton" name="genInvoice_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Invoice</string>
      </property>
     </widget>
     <widget class="QListView" name="contacts_listView">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>300</y>
       <width>281</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>291</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>280</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Extras</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Contacts</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>21</y>
       <width>871</width>
       <height>311</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="add_contact_Button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Contact</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_contact_Button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Sent</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QPushButton" name="settingAddUser_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
    <widget class="QTableView" name="setting_tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>921</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="settingDelete_Button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingAddSubC_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Sub C.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCalendar</class>
   <extends>QCalendarWidget</extends>
   <header>myCalendar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
